lib/atscppapi/src/include/atscppapi/Transaction.h

```diff
@@ -30,7 +30,7 @@
 #include "atscppapi/shared_ptr.h"
 #include "atscppapi/ClientRequest.h"
 #include "atscppapi/Response.h"
-
+#include <../ts/apidefs.h>
```

Comment:
Hmmm. Let me think about that a bit. I have an idea on how to deal with this but I need to flesh it out.